addons/account/views/account_move_views.xml

```diff
@@ -1566,6 +1590,43 @@
                         domain="[('my_activity_date_deadline', '=', context_today().strftime('%Y-%m-%d'))]"/>
                     <filter invisible="1" string="Future Activities" name="activities_upcoming_all"
                         domain="[('my_activity_date_deadline', '&gt;', context_today().strftime('%Y-%m-%d'))]"/>
+                    <filter name="late_invoices"
+                            string="Late Invoices"
+                            invisible="1"
```

Comment:
filters with invisible=1 that are selected by the mean of the context... theoretically, I've no idea how that behaves :sweat_smile: I suppose it's unavailable in the filter drop down, for the user to select it... but does it show in the filter bar when active?

Why not choosing the different _existing_ filters, available all the time that would allow to apply the exact same domain?? Because there's already the filter `late` (Overdue) that seems to do exactly that... :suspect:

Reply:
Yep making it invisible is just to not be displayed in the drop down. 
The reason i made new filters is because when adding multiple search_default, it uses an "or" in some cases and we want an "and", i will recheck to be 100% sure

Reply:
about this, Hugo is currently working on this matter also https://github.com/odoo/odoo/pull/178517 so i will reuse his code and when he is done, i will rebase and finish the task :smile: